counterexamples/transportation/segment/road/restrictions/bad-access-mode-not.yaml

```diff
@@ -22,5 +22,5 @@ properties:
     flags: [isLink, isTunnel, isPrivate]
     restrictions:
       access:
-        - allowed:
-            when: {modeNot: [foo]}
+        - accessType: allowed
+          when: {modeNot: [foo]}
```

Comment:
This is absolutely going in the right direction, as we need to simplify in the way you're aiming.

My only concern is that I don't love `accessType` as a property name.

1. I feel that it reads a bit awkwardly.
2. I feel that the `type` suffix doesn't really add a lot of information that's not already there.

Here are some alternatives we could consider:

1. In some places, *e.g.* flag rules, I *think* we are using the word `value` or `values`. Maybe `value` is enough, and has the benefit of consistency?
2. Alternatively, we could rename the parent `access` to `accessRestrictions` and use the word `access` instead of `accessType`?